src/parseWithPointers.ts

```diff
@@ -0,0 +1,35 @@
+import { Processor } from 'unified';
+import { IParserResult } from '@stoplight/types';
+import { ParseOpts } from 'remark-parse';
+
+import { parse } from './parse';
+import * as Mdast from './ast-types/mdast';
+import { IPosition } from './ast-types/unist';
+import * as TUnist from './ast-types/unist';
+
+interface Pointers {
+  [key: string]: IPosition
+}
+
+const getPointers = (node: Partial<TUnist.IParent>, pointers: Pointers = {}, path: string = '/'): Pointers  => {
```

Comment:
Should return [IParserResultPointers](https://github.com/stoplightio/types/blob/master/src/parsers.ts#L23).